collector/dmi.go

```diff
@@ -78,7 +79,7 @@ func NewDMICollector(logger log.Logger) (Collector, error) {
 	} {
 		if value != nil {
 			labels = append(labels, label)
-			values = append(values, *value)
+			values = append(values, strings.ToValidUTF8(*value, "�"))
```

Comment:
We need a test case for this. Best would be to have the real world output from the bug report for this.

Reply:
I added the quoted invalid UTF-8 string from the bug report as a fixture.